src/browser/monkey-menu.html

```diff
@@ -118,9 +118,9 @@
 
     <hr>
 
-    <p rv-if="activeScript.description">{'description'|i18n}
+    <p rv-if="activeScript.description"><strong>{'description'|i18n}</strong>
```

Comment:
I'm not sure how you picked this HTML style, but I'd greatly prefer:

<p rv-if="activeScript.description">
  <strong>{'description'|i18n}</strong> {activeScript.description}
</p>

(Maybe a linebreak before `{` if that's >80 chars.

Reply:
No, it was strong (copy/paste and edit of what was there), either is fine.  Bah markdown for eating (some) HTML tags.

Reply:
Your decision: &lt;b&gt; or &lt;strong&gt;, both are okay. I've started with &lt;u&gt; (which was really okay, but one should probably avoid underline here). The install dialog used &lt;strong&gt;. Current implementation is &lt;b&gt;.